api/app/hfi/__init__.py

```diff
@@ -1 +1,2 @@
 """ Module for HFI calculations """
+from .hfi_calc import hydrate_fire_centres, calculate_latest_hfi_results
```

Comment:
relative import? tsk tsk

Reply:
Elaborate? I'm more concerned with even having the imports here - it's just that `from app.hfi.hfi_calc import hydrate_fire_centres` in other modules seems excessive.

Reply:
What the relative dislike of relative imports?

Reply:
I thought since day 1 of this project we'd sworn off relative imports entirely

Reply:
I know for frontend it's hard to make refactors when you have relative imports. Not sure about python.

Reply:
If you're going to make me type the additional 7 characters I'll do it.

Reply:
I don't think I was around when the discussion about relative imports happened. It was back when I was only on this project half-time, so I don't know the rationale. But it's something that we've continued ever since

Reply:
Pffft. Doing something because we've always done it that way.